cid/helpers/quicksight/__init__.py

```diff
@@ -1029,6 +1030,53 @@ def update_dataset(self, definition: dict) -> Dataset:
         return self.describe_dataset(dataset_id)
 
 
+    def ensure_dataset_refresh_schedule(self, dataset_id, schedule_id):
+        """ Ensure that dataset has scheduled refresh """
+        try:
+            refresh_schedules = self.client.list_refresh_schedules(
+                AwsAccountId=self.account_id,
+                DataSetId=dataset_id
+            )
+            if "RefreshSchedules" in refresh_schedules and len(refresh_schedules["RefreshSchedules"]) > 0:
+                logger.info(f'Dataset {dataset_id} already has scheduled refresh.')
+                return
+        except self.client.exceptions.ResourceNotFoundException:
+            logger.info(f'DataSource {dataset_id} does not exist')
+            return
+        except self.client.exceptions.AccessDeniedException:
+            logger.info(f'No quicksight:ListDataSets permission or missing DataSetId {dataset_id}')
+            return
+        except Exception as e:
+            logger.info(f'Unable to list refresh schedules for dataset {dataset_id}.')
+            logger.info(e)
+            return
+
+        schedule = {
```

Comment:
I would move defintion partialy to the dataset defintion. We already have some cases when we need Incremental

Reply:
But not all of it

Reply:
Should we move it resources.yaml to have a high level object
```
 schedules: 
         default:  
               <definition>
```
 and then name of the schedule as definition of each dataset like 
 
```
 dataset:
         schedule: default
```
this would allow to export schedule as a part of export function